test/integration/suite_test.go

```diff
@@ -75,6 +77,7 @@ func TestControllers(t *testing.T) {
 
 var _ = BeforeSuite(func() {
 	logf.SetLogger(zap.New(zap.WriteTo(GinkgoWriter), zap.UseDevMode(true)))
+	format.MaxLength = 20000
```

Comment:
Can you add a comment for what this does?

Reply:
added in a8ed93cd59d7d55ed973e046463ea7b59a410b51